app/code/Magento/Checkout/Test/Mftf/ActionGroup/StorefrontOpenCheckoutCartPageActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontOpenCheckoutCartPageActionGroup">
+        <annotations>
+            <description>Goes to the Storefront Checkout Cart page.</description>
+        </annotations>
+
+        <amOnPage url="{{CheckoutPage.url}}/cart" stepKey="openCheckoutCartPage"/>
+        <waitForPageLoad stepKey="waitForCheckoutCartPageLoaded"/>
+        <amOnPage url="{{CheckoutPage.url}}" stepKey="openCheckoutPage"/>
+        <waitForPageLoad stepKey="waitForCheckoutPageLoaded"/>
```

Comment:
What's the reason of going to cart and after that to checkout page?
I'd use the following Action Group instead `StorefrontOpenCheckoutPageActionGroup`.

Reply:
@eduard13 bug reproduce only when customer add items to cart and reach checkout , use a unregistered e-mail in e-mail field and fill address data after that visits any other page as long outside from checkout page and after that come back to checkout.  

So i decided to take this case into consideration. If your opinion is opposite i can remove this case.

Reply:
If I'm not mistaken, the issue comes up after navigating from checkout to any other page, and then you're coming back to checkout. So there is no required step to be on cart page.
Correct me if I'm wrong.

Reply:
@eduard13 Yes you are right. So you recommend replace `StorefrontOpenCheckoutCartPageActionGroup` to `StorefrontOpenCheckoutPageActionGroup` ?

Reply:
Yes.

Reply:
@eduard13 done )